pom.xml

```diff
@@ -87,13 +87,13 @@
         <dependency>
             <groupId>gov.nasa.pds</groupId>
             <artifactId>registry-common</artifactId>
-            <version>1.2.0-SNAPSHOT</version>
+            <version>1.1.1</version>
         </dependency>
         <!-- PDS4 label / data table parser -->
         <dependency>
             <groupId>gov.nasa.pds</groupId>
             <artifactId>pds4-jparser</artifactId>
-            <version>2.2.0-SNAPSHOT</version>
+            <version>2.1.1</version>
```

Comment:
Same as above but in this case there was no log4j warning of sonatype, but the snapshot version available for jparser was 2.1.2-SNAPSHOT, I changed that in the jparser repo to 2.2.0-SNAPSHOT.